phpBB/language/en/ucp.php

```diff
@@ -38,23 +38,23 @@
 
 // Privacy policy and T&C
 $lang = array_merge($lang, array(
-	'TERMS_OF_USE_CONTENT'	=> 'By accessing “%1$s” (hereinafter “we”, “us”, “our”, “%1$s”, “%2$s”), you agree to be legally bound by the following terms. If you do not agree to be legally bound by all of the following terms then please do not access and/or use “%1$s”. We may change these at any time and we’ll do our utmost in informing you, though it would be prudent to review this regularly yourself as your continued usage of “%1$s” after changes mean you agree to be legally bound by these terms as they are updated and/or amended.<br />
-	<br />
-	Our forums are powered by phpBB (hereinafter “they”, “them”, “their”, “phpBB software”, “www.phpbb.com”, “phpBB Limited”, “phpBB Teams”) which is a bulletin board solution released under the “<a href="http://opensource.org/licenses/gpl-2.0.php">GNU General Public License v2</a>” (hereinafter “GPL”) and can be downloaded from <a href="https://www.phpbb.com/">www.phpbb.com</a>. The phpBB software only facilitates internet based discussions; phpBB Limited is not responsible for what we allow and/or disallow as permissible content and/or conduct. For further information about phpBB, please see: <a href="https://www.phpbb.com/">https://www.phpbb.com/</a>.<br />
-	<br />
+	'TERMS_OF_USE_CONTENT'	=> '<p>By accessing “%1$s” (hereinafter “we”, “us”, “our”, “%1$s”, “%2$s”), you agree to be legally bound by the following terms. If you do not agree to be legally bound by all of the following terms then please do not access and/or use “%1$s”. We may change these at any time and we’ll do our utmost in informing you, though it would be prudent to review this regularly yourself as your continued usage of “%1$s” after changes mean you agree to be legally bound by these terms as they are updated and/or amended.
```

Comment:
why is there a p tag that should be in the template? and why are we doubling the br? just use one. We can fix it in the template with an improvement to the `lang()` function in the future to convert them to proper p tags with classes.

Reply:
to the p-tag: Because one of the aims of this PR is, that this p-tag wrapping around `AGREEMENT_TEXT`, `L_TERMS_OF_USE` and `L_COPPA_BIRTHDAY` gets removed. 
to the 2x br: If i should not use a p-tag for every paragraph, then i need to somehow create this empty line between each paragraph. If not, this will get a real wall of text. 
to the future: What do you mean? Within this PR or not? If not, than nothing is against my current solution (or the solution before), we can undo it in the future, too, if we improve the `lang()`